modules/sync/syncs/urlsynchronization.h

```diff
@@ -93,6 +102,9 @@ class UrlSynchronization : public ResourceSynchronization {
 
     // The thread that will be doing the synchronization
     std::thread _syncThread;
+
+    /// Determines how long the file is valid before it should be downloaded again.
+    double _secondsUntilResync;
```

Comment:
```suggestion
    double _secondsUntilResync = 0.0;
```

just feeling anxious about uninitialized variables ;)